src/components/Causes/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { CausesSection, CausesPhoto, CausesTextSection, CausesH1, CausesH2, CausesP } from './css';
+import { Causes } from './data';
+
+const CausesWeChampion = () => {
```

Comment:
Consider destructuring here
aka

const { headline, headline2, point1, point2, point3 } = Causes;

then sticking in < CausesH1 > { headline } </ CausesH1 > for example.
It just makes sticking in the variables into the JSX a bit cleaner :) Not 100% necessary all the time though.

would also make the { Causes } from data lowercase for the first letter since it's a variable :)